lib/DependencyInjection/Compiler/SerializerContextConfigurationPass.php

```diff
@@ -18,7 +18,10 @@ public function process(ContainerBuilder $container): void
         if ($container->hasExtension('framework')) {
             /** @var FrameworkExtension $framework */
             $framework = $container->getExtension('framework');
+            $resolvingBag = $container->getParameterBag();
+
             $frameworkConfigs = $container->getExtensionConfig('framework');
+            $frameworkConfigs = $resolvingBag->resolveValue($frameworkConfigs);
```

Comment:
Probably `$container->resolveEnvPlaceholders` should be used here, as used in CheckDependencyPass.php:L20

Reply:
Hi @alekitto,  
I replaced with $container->resolveEnvPlaceholders and rebase upon the latest master commit

Reply:
I still see a merge commit, probably something went wrong with the rebase.